src/Strings.cpp

```diff
@@ -168,3 +169,12 @@ std::string Strings::getKey( int key ) {
 std::string Strings::getLevelScoreString(int level, int score) {
     return "Your score for level " + std::to_string(level+1) + " was: " + std::to_string(score) + "\n";
 }
+
+std::string Strings::getFinalScoreString(std::vector<unsigned int> scores) {
```

Comment:
I think it would be better to move the actual summing of the scores into HumanView, that way it can call getScore instead of having to pass a whole vector of all the scores. Then just have a string in the strings map in Strings that is "Your final score is " and concatenate that to the summed scores in HumanView. Then you can get rid of getScoresVector in GameLogic